script/Arcueid/Day 10/10_00_ARC10LIVING.txt

```diff
@@ -173,12 +180,15 @@ I sit up from the bed and try my best to put on a natural-looking smile.
 [sha:e8cedd77878e6115fd15e7e033eec4135ede84ba]{
 -- Page 897, Offset 26864.
 -- 「そうかな、俺はすごくありがたいけど。目覚まし時計の音なんかより、何倍もあったかい感じがする」
-"That so? But I %{i}am%{/i} extremely grateful. Compared with the sound of an alarm clock, this feels much warmer."
+"That so? But I %{i}am%{/i} extremely grateful.
+ This is a much warmer wake-up call than the blaring of an alarm clock."
 }
 [sha:44a37e56a9f77c20befc60f9ed247707559b082e]{
 -- Page 898, Offset 26865.
 -- 今朝もそのおかげで目が覚めた。
-Thanks to that, I awake to another day.
+// This is one of those sentences that's so simple but also feels so hard to TL.
+// Still don't love it.
+I'm lucky to have been woken this way again today.
```

Comment:
with just lit.
> (今朝も (そのおかげで) ) 目が覚めた。
thanks to [hisui waking me up], today too, I awaken.

"lucky" and "wake" in a sentence together give a sense that there is some explicit alternative (typically I'd assume death). And given he _was_ in mortal danger the night before and pushed himself hard, I would definitely understand this as being grateful for being alive, not for Hisui's considerately waking him up.

Reply:
Yeah, it just feels like such a weird thing to say literally that I got hung up on it. /shrug

Reply:
maybe just "Thanks to that warmth, I greet another day" or something like that